kube-controllers/cmd/kube-controllers/main.go

```diff
@@ -261,9 +266,10 @@ func runHealthChecks(ctx context.Context, s *status.Status, k8sClientset *kubern
 			// carry on
 		}
 
-		// skip healthchecks if configured
-		// Datastore HealthCheck
-		healthCtx, cancel := context.WithTimeout(ctx, 10*time.Second)
```

Comment:
Maybe we should have two different timeouts - EnsureInitialized can potentially make a few HTTP calls under the hood.